go/test/endtoend/cluster/cluster_util.go

```diff
@@ -74,11 +74,13 @@ func GetMasterPosition(t *testing.T, vttablet Vttablet, hostname string) (string
 }
 
 // VerifyRowsInTabletForTable Verify total number of rows in a table
+// this is used to check replication caught up the changes from master
 func VerifyRowsInTabletForTable(t *testing.T, vttablet *Vttablet, ksName string, expectedRows int, tableName string) {
 	timeout := time.Now().Add(10 * time.Second)
 	for time.Now().Before(timeout) {
-		qr, err := vttablet.VttabletProcess.QueryTablet("select * from "+tableName, ksName, true)
-		require.Nil(t, err)
+		// ignoring the error check, if the newly created table is not replicated, then there might be error and we should ignore it
+		// but eventually it will catch up and if not caught up in required time, testcase will fail
+		qr, _ := vttablet.VttabletProcess.QueryTablet("select * from "+tableName, ksName, true)
 		if len(qr.Rows) == expectedRows {
```

Comment:
should check for qr nil before accessing Rows. If there is an error, qr will be nil.

Reply:
Added nil check.